kits/rosie/arm.hpp

```diff
@@ -0,0 +1,31 @@
+#include "robot_model.hpp"
+#include "trajectory.hpp"
+
+using namespace hebi;
+
+class GripperArm{
+  public:
+  std::unique_ptr<robot_model::RobotModel> model;
```

Comment:
Make these private and encapsulate them

Reply:
I also thought about using a struct to avoid like 10 different getters

Reply:
Yeah... I understand your perspective here. There are a few solutions to this, but I don't think that making all fields public is the right way forward here.

Reply:
okay I'll just make them properly encapsulated the classes (if we want to keep them)